app/controllers/subscription.js

```diff
@@ -205,13 +207,18 @@ export default Ember.Controller.extend({
       return;
     }
 
-    if (["create","update"].contains(data.operation)) {
+    if (["create","update"].includes(data.operation)) {
```

Comment:
I think `indexOf` should be used here, for compatibility with I.E 😆

Reply:
i changed it to `includes` due to console warning. should i use `indexOf` or change it back to `contains` ?

Reply:
Use `indexOf`, considering I.E.